cl/corpus_importer/utils.py

```diff
@@ -87,9 +198,9 @@ def match_lists(
     :return: Matches if found or False
     """
     # We import this here to avoid a circular import
-    from cl.corpus_importer.management.commands.harvard_opinions import (
-        compare_documents,
-    )
+    # from cl.corpus_importer.management.commands.harvard_opinions import (
+    #     compare_documents,
+    # )
```

Comment:
What is going on here?

Reply:
I forgot to delete this, I moved some functions at corpus_importer users